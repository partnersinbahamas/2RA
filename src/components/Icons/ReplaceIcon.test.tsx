import React from 'react';
import { render } from '@testing-library/react';
import ReplaceIcon from './ReplaceIcon';

test('ReplaceIcon', () => {
  const { container } = render(<ReplaceIcon />);
  expect(container).toMatchSnapshot();
});
