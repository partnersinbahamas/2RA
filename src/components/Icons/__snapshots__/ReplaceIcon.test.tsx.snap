// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReplaceIcon 1`] = `
<div>
  <svg
    class="sc-beySPh jroDkm"
    viewBox="0 0 15 18"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M0 8.92349C0 6.24049 1.734 4.18049 4.416 3.46049L3.435 2.83949C3.30024 2.73532 3.19159 2.60121 3.11765 2.44778C3.0437 2.29434 3.00651 2.12579 3.009 1.95549C3.009 1.39949 3.435 0.974487 3.991 0.974487C4.187 0.974487 4.383 1.03949 4.547 1.13749L7.557 3.16649C7.785 3.29649 8.014 3.59149 8.014 3.98349C8.01033 4.18006 7.95404 4.37205 7.851 4.53949L5.822 7.54949C5.7246 7.69332 5.59282 7.81055 5.43862 7.89053C5.28441 7.97051 5.11268 8.01072 4.939 8.00749C4.81195 8.00634 4.68641 7.9799 4.56968 7.92972C4.45296 7.87953 4.3474 7.80661 4.25916 7.7152C4.17091 7.6238 4.10176 7.51573 4.05572 7.39731C4.00968 7.2789 3.98768 7.1525 3.991 7.02549C3.991 6.79649 4.056 6.60049 4.154 6.43749L4.841 5.42349C3.173 5.75049 1.995 7.22249 1.995 8.92349C1.995 10.1985 2.617 11.4095 3.598 12.1935C3.72374 12.2888 3.82496 12.4128 3.8933 12.555C3.96164 12.6973 3.99513 12.8537 3.991 13.0115C3.99047 13.2718 3.88684 13.5212 3.7028 13.7053C3.51875 13.8893 3.26928 13.993 3.009 13.9935C2.78 13.9935 2.551 13.9275 2.388 13.7965C0.883 12.5865 0 10.8205 0 8.92449V8.92349ZM14.014 9.07749C14.014 11.7595 12.28 13.8205 9.598 14.5395L10.579 15.1615C10.874 15.3905 11.037 15.7175 11.037 16.0445C11.037 16.6005 10.579 17.0255 10.023 17.0255C9.827 17.0255 9.631 16.9605 9.467 16.8625L6.457 14.8345C6.31649 14.7494 6.20049 14.6293 6.12037 14.4859C6.04026 14.3425 5.99878 14.1807 6 14.0165C6 13.7875 6.065 13.6245 6.164 13.4605L8.192 10.4505C8.388 10.1565 8.715 9.99349 9.075 9.99349C9.565 9.99349 10.056 10.3855 10.056 10.9745C10.056 11.1705 9.958 11.3995 9.86 11.5635L9.173 12.5775C10.841 12.2505 12.019 10.7775 12.019 9.07749C12.019 7.80149 11.397 6.59149 10.416 5.77349C10.154 5.61049 10.023 5.28349 10.023 4.98849C10.023 4.49849 10.416 3.97449 11.037 3.97449C11.234 3.97449 11.463 4.07249 11.627 4.20349C13.163 5.41349 14.014 7.18049 14.014 9.07749Z"
      fill="#282d37"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
